tests/ui/specs/LabeledIconButton/LabeledIconButtonPage.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+const {getSubComponent, getText, Page} = require('@enact/ui-test-utils/utils');
+
+const getMarqueeText = getSubComponent({lib: 'ui', component: 'Marquee', child: 'text'});
+
+class LabeledIconButtonInterface {
+	constructor (id) {
+		this.id = id;
+		this.selector = `#${this.id}`;
+	}
+
+	focus () {
+		return browser.execute((el) => el.focus(), $(`#${this.id}`));
+	}
+	hover () {
+		return $(this.selector).moveTo({xOffset: 0, yOffset: 0});
+	}
+
+	get self () {
+		return $(this.selector);
+	}
+	get valueText () {
+		return getText(getMarqueeText(this.self));
+	}
+	iconValue () {
+		return getText($(this.selector + ' .Icon_Icon_icon')).codePointAt();
+	}
+	isSelected () {
+		return $(this.selector + ' .Button_Button_selected').isExisting();
```

Comment:
I think we have `hasClass` from `ui-test-utils` like below.
```
const {hasClass} = require('@enact/ui-test-utils/utils');
...
get isIconAfter () {
	return hasClass('iconAfter', this.self);
}
```
Please find more in sandstone/tests/ui/specs/Button/Button-specs.js

Reply:
As I tried applying your suggestion, I've noticed that because of the tree structure of this component, the selected className is applying to the `Button` (as 'Button_Button_selected') which is a grand-child of `LabeledIconButton`. So:
```
isSelected () {
	return hasClass('Button_Button_selected', this.self);
}
```
or
```
isSelected () {
	return hasClass('selected', this.self);
}
```
would result in `false`.
So, I think for now this should be left as it is. What do you think?

Reply:
I see!